frontend/src/components/pages/EventEditPage.vue

```diff
@@ -69,3 +68,17 @@ watch(
   { immediate: true, flush: "post" }
 );
 </script>
+
+<style scoped>
+.flex {
+  display: flex;
+}
+
+.flex-col {
+  flex-direction: column;
+}
+
+.spinner {
+  margin: auto;
+}
+</style>
```

Comment:
Vuetify has a bunch of styles/classes
https://next.vuetifyjs.com/en/styles/flex/